src/pybind/mgr/dashboard/frontend/src/styles.scss

```diff
@@ -365,13 +365,23 @@ h3.page-header {
   border-color: #f0f0f0;
 }
 
+/* Tooltips */
 .tooltip-wide .tooltip-inner {
   width: 400px;
 }
-
 .tooltip-inner {
   background-color: white;
   border: 1px solid grey;
   color: #333;
   font-size: 1.1em;
 }
+
+/* Notifications */
+.toast-message .custom-icon {
+  /* Display icons in white, regardless of their origin color. */
+  filter: brightness(0) invert(1);
+  -webkit-filter: brightness(0) invert(1);
+  -moz-filter: brightness(0) invert(1);
+  -o-filter: brightness(0) invert(1);
+  -ms-filter: brightness(0) invert(1);
```

Comment:
It seems that without prefix `filter` [covers 90% of the users](https://caniuse.com/#feat=css-filters) and if you check the usual desktop browsers it's almost 100%. For the 0-10% remaining, the effect would just be a red-coloured logo instead, so not a big deal. I was wondering if the common practice (web-dev rookie here) is to generate new UI assets via CSS filters or process and save them as new files.

If we want to generate cross-browser CSS, why not using some library? Just googled and found these:
- https://github.com/Schepp/CSS-Filters-Polyfill
- https://github.com/pascalduez/SassyFilters
- https://github.com/iamvdo/pleeease-filters

Reply:
Angular CLI already does that: https://angular.io/guide/build#configuring-browser-compatibility.
I try to remove all the `filter` styles with prefixes and the CLI automatically added the `webkit-filter`.

We could remove the filters here, but I'm guessing we have lots of prefixes throughout the rest of the code.
A separate PR would be my recommendation, and we could also change the settings to get more coverage.

Reply:
I think using a library to handle the filters is too much for something really trivial. IMHO, if there are prefixes that are not used anymore or only for a small number of supported browser, so what.

Reply:
Thanks, @votdev.